Tella/Data/Networking/APICall/APIError.swift

```diff
@@ -39,6 +41,8 @@ extension APIError: LocalizedError {
             return customNcErrorMessage(errorCode: code)
         case .errorOccured :
             return LocalizableError.commonError.localized
+        case .dropboxApiError(let error):
+            return DropboxErrorParser.customDropboxErrorMessage(error: error)
```

Comment:
What do you think of using an `APIError`  extension that contains `customDropboxErrorMessage` function ?